pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -425,9 +443,16 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 		case <-domainCtx.publishTicker.C:
 			publishProcessesHandler(&domainCtx)
 
+		case change := <-subEdgeNodeInfo.MsgChan():
+			subEdgeNodeInfo.ProcessChange(change)
+			waitEdgeNodeInfo = false
+
 		case <-stillRunning.C:
 		}
 		ps.StillRunning(agentName, warningTime, errorTime)
+		if time.Since(wtTime) > 5*time.Minute { // wait for max of 5 minutes
```

Comment:
Why do you only want to wait for 5 minutes? Shouldn't zedagent always publish something even if there is no network connectivity to the controller for instance?